source/package.template.html.erb

```diff
@@ -32,6 +32,19 @@
                   <%= package.namespace %>
                 </a>
               </em><br/>
+              <% if is_valid_github_url(version._source.url) %>
```

Comment:
This check is awesome.

I am still a little bit worried about a case where the full path of `version._source.url` isn't defined. But further down the code, it's already accessing that same reference without any checks, so we'll let it fly:
```
(<a href="<%= version._source.url %>">View on GitHub</a>)
```